google/cloud/bigtable/examples/README.md

```diff
@@ -3,25 +3,37 @@
 This directory contains minimal examples to get started quickly, and are
 referenced from the Doxygen landing page as "Hello World".
 
-## Compiling the Examples
+## Table of Contents
+
+- [Before you begin](#before-you-begin)
+  - [Compiling the Examples](#compiling-the-examples)
+  - [Running the examples against the Cloud Bigtable Emulator](#running-the-examples-against-the-cloud-bigtable-emulator)
+  - [Running the examples against a production version of Cloud Bigtable](#running-the-examples-against-a-production-version-of-cloud-bigtable)
+- [Samples](#samples)
+  - [Hello World](#hello-world)
+  - [Administer Instances](#administer-instances)
+
+## Before you begin
+
+### Compiling the Examples
 
 These examples are compiled as part of the build for the Cloud Bigtable C++
 Client.  The instructions on how to compile the code are in the
 [top-level README](../../README.md) file.
 
-## Running the examples against the Cloud Bigtable Emulator
+### Running the examples against the Cloud Bigtable Emulator
 
 The easiest way to run these examples is to use the Cloud Bigtable Emulator.
```

Comment:
So this is no longer true when referring to the instance admin example, right?  It gets complicated because you need to run two different emulators.  Maybe we should remove the emulator section altogether?  @sduskis thoughts?  What do we do for other client libraries?